Makefile

```diff
@@ -48,7 +48,7 @@ anvil_start:
 
 anvil_start_with_block_time:
 	@echo "Starting Anvil..."
-	anvil --load-state contracts/scripts/anvil/state/alignedlayer-deployed-anvil-state.json --block-time 3
+	anvil --load-state contracts/scripts/anvil/state/alignedlayer-deployed-anvil-state.json --block-time 6
```

Comment:
Why the block time has changed?

Reply:
3 seconds block time was too low for some test cases in this PR.
This change shouldn't have been pushed though, reverted to previous version 👍